src/utility.js

```diff
@@ -200,3 +204,54 @@ function countFlags(value) {
 	}
 	return count;
 }
+
+/**
+ * Compute compensating position offsets if body or html element has non-static position attribute.
+ * @private
+ * @param {number} windowWidth Window width in pixels.
+ * @param {number} windowHeight Window height in pixels.
+ * @return {Offsets} The top, left, right, bottom offset in pixels
+ */
+function computePositionCompensation(windowWidth, windowHeight) {
+	// Check if the element is "positioned". A "positioned" element has a CSS
+	// position value other than static. Whether the element is positioned is
+	// relevant because absolutely positioned elements are positioned relative
+	// to the first positioned ancestor rather than relative to the doc origin.
+	var isPositioned = function(el) {
```

Comment:
I'd like to see this function pulled out of the `computePositionCompensation()` function. This has a couple benefits:

1. (Big one) It can be unit tested on it's own.
2. It makes sure that the function is only generated once at run time.

Just move it to be another utility function in this file.